Meadowlark-js/tests/e2e/automation-setup/e2e.env

```
@@ -11,12 +11,14 @@ MEADOWLARK_STAGE=local
 LOG_LEVEL=WARN
 IS_LOCAL=true
 
+POSTGRES_PORT=5433
+
 OAUTH_SERVER_ENDPOINT_FOR_OWN_TOKEN_REQUEST=http://localhost:3000/local/oauth/token
 OAUTH_SERVER_ENDPOINT_FOR_TOKEN_VERIFICATION=http://localhost:3000/local/oauth/verify
 
 OWN_OAUTH_CLIENT_ID_FOR_CLIENT_AUTH=meadowlark_verify-only_key_1
 OWN_OAUTH_CLIENT_SECRET_FOR_CLIENT_AUTH=meadowlark_verify-only_secret_1
+OAUTH_HARD_CODED_CREDENTIALS_ENABLED=true
 
 # End To End
-ROOT_URL = http://localhost:3000
-OAUTH_HARD_CODED_CREDENTIALS_ENABLED=true
\ No newline at end of file
+ROOT_URL = http://localhost:3000
```

There is a backdoor mechanism for changing which database name to use in PostgreSQL, which I had forgotten about but rediscovered. Let's see if this helps, so that the service does not complain about the meadowlark database not existing.

```suggestion
ROOT_URL = http://localhost:3000
MEADOWLARK_DATABASE_NAME=postgres
```